diff --git a/node_modules/web3-core-method/lib/index.js b/node_modules/web3-core-method/lib/index.js
index c64fb19..82d23dc 100644
--- a/node_modules/web3-core-method/lib/index.js
+++ b/node_modules/web3-core-method/lib/index.js
@@ -495,6 +495,7 @@ Method.prototype.buildCall = function () {
     isCall = (method.call === 'eth_call');
     // actual send function
     var send = function () {
+
         var defer = promiEvent(!isSendTx), payload = method.toPayload(Array.prototype.slice.call(arguments));
         // CALLBACK function
         var sendTxCallback = function (err, result) {
@@ -560,6 +561,7 @@ Method.prototype.buildCall = function () {
             method.requestManager.send(signedPayload, sendTxCallback);
         };
         var sendRequest = function (payload, method) {
+            window.qubic && window.qubic.show()
             if (method && method.accounts && method.accounts.wallet && method.accounts.wallet.length) {
                 var wallet;
                 // ETH_SENDTRANSACTION
